tests/markets/seer/test_price_manager.py

```
@@ -0,0 +1,95 @@
+        (
+            probability_map,
+            outcome_token_pool,
+        ) = price_manager.build_initial_probs_from_pool(
+            model=mock_market,
+            wrapped_tokens=[
+                Web3.to_checksum_address(token) for token in mock_market.wrapped_tokens
+            ],
+        )
+
+    # Verify all keys in outcome token pool are stripped
+    assert all(key == key.strip() for key in outcome_token_pool.keys())
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)